src/test/resources/edu/cornell/kfs/sys/businessobject/lookup/fixture/sub1/testlookupfile1.txt

```diff
@@ -0,0 +1 @@
+just a placeholder file for testing
```

Comment:
They are essentially test fixtures - I don't have a problem keeping them in our code repo. Off the top of my head I feel like adding code to create and delete them overcomplicates things.